### PR TITLE
Lesson four

### PR DESCRIPTION
**Pull request by Sathiyakugan**

### CODE REVIEW

**Comment by mezubeen** [comment]
same issue "No earthquake found" even in the latest commit

**Comment by Kvaibhav01** [comment]
Change `http` to `https` in the _EarthquakeActivity.java_ file from where the details are fetched.

**Review by Kvaibhav01 — APPROVED** [review]

**Review by Kvaibhav01 — APPROVED** [review]

File: app/src/main/java/com/example/android/quakereport/EarthquakeActivity.java [hunk 1]
```diff
@@ -182,12 +182,13 @@ public void onLoadFinished(Loader<List<Earthquake>> loader, List<Earthquake> ear
...
-        mAdapter.clear();
-
+        //mAdapter.clear();
+        
         // If there is a valid list of {@link Earthquake}s, then add them to the adapter's
         // data set. This will trigger the ListView to update.
         if (earthquakes != null && !earthquakes.isEmpty()) {
-            mAdapter.addAll(earthquakes);
+            //mAdapter.addAll(earthquakes);
+            updateUi(earthquakes);
```
Comment (amansharma-dev) [inline app/src/main/java/com/example/android/quakereport/EarthquakeActivity.java]: i can not able to solve updateUi(earthquakes); error  that says .... Cannot resolve method 'updateUi(java.util.List<com.example.android.quakereport.Earthquake>)'

and android studio says create method for updateUi .. how and what to pass ??

Comment (Kvaibhav01) [reply app/src/main/java/com/example/android/quakereport/EarthquakeActivity.java]: You are calling the `updateUi` method which doesn't exists in your code yet. Try to create the same and resolve the error.

Comment (rr47061) [reply app/src/main/java/com/example/android/quakereport/EarthquakeActivity.java]: use this code: 
mAdapter.addAll(earthquakes);
            //updateUi(earthquakes);

**Comment by SudKul** [comment]
We have upgraded the code in the **lesson-four** branch to AndroidX in this [commit](https://github.com/udacity/ud843-QuakeReport/commit/5685f5c1881b738a394514abedc48dce0f2ad342). We expect students to replicate the similar updates to other branches, where needed.

**Pull request closed without merging**